### ISSUE TITLE
Can't create folder in codesanbox ??

### ISSUE THREAD

**Issue by sahilrajput03** [codesandbox/codesandbox-client#5664]
Please this is a major bug so i am just putting this gif to demonstrate the issue.. 
Though i am using firefox browser version 87.0 64 bit on ubuntu 20.04.

Check [it here](https://i.imgur.com/Jb3w7wB.mp4).

**Comment by sannek** [comment]
Hi,

Thanks for your report - this is indeed a big bug! We're going to try and fix it asap, but it seems to be related to spaces in any higher level folders. I'll report back here when a fix is deployed - hopefully tonight (CEST), but until then removing spaces from your higher-level folder names should allow you to create subfolders again.

Apologies for the inconvenience!

**Comment by sahilrajput03** [comment]
I can't rename a folder, why is that..? :eyes: 
See [it here in my demonstration video](https://youtu.be/T5DZfm6SHLQ).

**Comment by sannek** [comment]
Wow that is really odd - and  I can quite easily reproduce it. 
Your previous issue (creating the folders) should be fixed, but I will have a look at this one. It doesn't seem to raise an error when trying to rename a folder, but somehow it doesn't persist in the database. 

Thanks for reporting!

**Comment by sahilrajput03** [comment]
Wow, I can now create folder in that folder, Yikes!! :heart: , Thanks for fixing this. Thanks to codesandbox team, yikes!

**Comment by sahilrajput03** [comment]
But i still can't rename any folder..!

**Comment by sannek** [comment]
Yeah! That one is going on my "fix today" list- sorry! 😬  I think we might have introduced a new bug when fixing the old bug...

**Comment by sahilrajput03** [comment]
Hmm., i appreciate your work! I hope you and team will fix this as well in some time. 
<3

**Comment by lbogdan** [comment]
@sannek Any updates on this?

**Comment by sahilrajput03** [comment]
> @sannek Any updates on this?

Yes, the folder name updating is still not working..!!

Check it live at : https://youtu.be/-FlA0nSz_8g

**Comment by sannek** [comment]
Apologies! It slipped through all the other things. I'm adding it to the server issues right now and will pick it up this week!

**Comment by sannek** [comment]
Just FYI - I am working on this, but it is turning out to be a little weirder / more complicate than I thought. So I haven't forgotten!

**Comment by sahilrajput03** [comment]
Any update on this issue..?
